### PR TITLE
Feat: Added home page

### PR DESCRIPTION
**Pull request by kiranpratyush**
PR for Home page

[Testing link](https://testing2364.netlify.app/)

### CODE REVIEW

**Review by himadri2110 — APPROVED** [review]
Good to go!

File: HTML/CSS/HOMEPAGE/style.css [hunk 1]
```diff
@@ -0,0 +1,32 @@
+body {
+  background: var(--color-background);
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+}
```
Comment (himadri2110) [inline HTML/CSS/HOMEPAGE/style.css:6]: Suggestion: You can use shorthand for `background` property here.

**Review by msdeshmukh009 — APPROVED** [review]
Looking good!!

**Comment by kiranpratyush** [comment]
> Looking good!!

Thank you

**Pull request merged**